client/components/tinymce/plugins/simple-payments/dialog/index.jsx

```diff
@@ -27,27 +26,34 @@ class SimplePaymentsDialog extends Component {
 		isEdit: PropTypes.bool.isRequired,
 		onChangeTabs: PropTypes.func.isRequired,
 		onClose: PropTypes.func.isRequired,
+		onInsert: PropTypes.func.isRequired,
```

Comment:
Oh, I had no idea that the dialog is used somewhere else than `tinymce/plugins` 😦 Thanks!